Fix grammar and capitalization (#630)
preactjs/preact-www: content/en/guide/v10/getting-started.md

}
```

> [Babel](https://babeljs.io/) has some of the best documentation out there. We highly recommend checking it out for questions surrounding Babel and how to set it up.

### Aliasing React to Preact

At some point, you'll probably want to make use of the vast React ecosystem. Libraries and Components originally written for React work seamlessly with our compatibility layer. To make use of it, we need to point all `react` and `react-dom` imports to Preact. This step is called _aliasing._

> #### Aliasing in Webpack

Reviewer: [they're picky about this one](https://webpack.js.org/branding/#the-name) :P

```suggestion
#### Aliasing in webpack
```

> To alias any package in Webpack, you need to add the `resolve.alias` section

Reviewer: ```suggestion
To alias any package in webpack, you need to add the `resolve.alias` section
```

};
```

#### Aliasing in Jest

> Similar to bundlers, [Jest](https://jestjs.io/) allows to rewrite module paths. The syntax is a bit
> different than in Webpack, for example, because it's based on regex. Add this to your
> Jest configuration:

Reviewer: ```suggestion
[Jest](https://jestjs.io/) allows the rewriting of module paths similar to bundlers.
These rewrites are configured using regular expressions in your Jest configuration:
```
